src/themes/dspace/app/submission/sections/upload/file/section-upload-file.component.html

```diff
@@ -0,0 +1,59 @@
+<ng-container *ngIf="fileData">
+    <div>
+      <div class="py-2">
+        <div class="d-flex align-items-center border-2 border-bottom mb-2 pb-1">
+          <div class="d-flex flex-grow align-items-center">
+            <h3 class="m-0 text-primary">{{fileName}} <span class="text-muted">({{fileData?.sizeBytes | dsFileSize}})</span></h3>
+            <ng-container *ngIf="fileData.accessConditions && fileData.accessConditions.length">
+              <span class="ml-2" *ngIf="fileData.accessConditions[0].name == 'restricted'"><i class="fa fa-lock text-danger"></i></span>
+              <span class="ml-2" *ngIf="fileData.accessConditions[0].name == 'embargo'"><i class="fa fa-lock text-warning"></i></span>
+              <span class="ml-2" *ngIf="fileData.accessConditions[0].name == 'openaccess'"><i class="fa fa-lock-open text-success"></i></span>
+            </ng-container>
+          </div>
+          <div *ngIf="!readOnly" class="d-flex align-items-center action-list">
+            <ng-container>
```

Comment:
this container seems useless (no condition, no loop) --> could be removed (if it's your change)